### ISSUE TITLE
Search engine suggestions recommends redirection pages

### ISSUE THREAD

**Issue by samueldr** [nixos-users/wiki#28]
```
[17:21:09] <krey> When you type "Overlay" into search it suggests both "Overlay" and "Overlays"
[17:21:56] <krey> I don't really see the need for having both, even if one's just a redirect
[17:22:57] <samueldr> hmmm, that's something that won't affect only the overlay page, so it should be tackled at the source...
[17:23:03] <samueldr> which I don't know yet how to handle
```

**Comment by Mic92** [comment]
This cannot be fixed unless a different search backend is used: https://stackoverflow.com/a/26283774

**Comment by makefu** [comment]
If we want to edit (or remove) one of these pages this link can be used https://nixos.wiki/index.php?title=Overlay&action=edit

**Comment by Mic92** [comment]
If we would use a template for redirects, it would be possible to add `__NOINDEX__` instead: https://www.mediawiki.org/wiki/Help:Controlling_search_engine_indexing

**Comment by samueldr** [comment]
Yes, the issue (while terse) is about finding a way to make sure it isn't in the results :). I didn't have time to look into it at the moment it was reported.

I just added `__NOINDEX__` to the *Overlay* page, let's see if this is picked up.

**Comment by Mic92** [comment]
Seems to work: https://nixos.wiki/index.php?search=Applying+overlays+automatically&go=Go

**Comment by Mic92** [comment]
No it does not: https://nixos.wiki/index.php?search=overlay&title=Special%3ASearch&profile=default&fulltext=1

**Comment by Mic92** [comment]
The completion is pretty dump. It ranks a `chapter` higher then an article.

**Comment by samueldr** [comment]
Hmmm, while it may work for the search, I think this is what this was about:

![20180430180547](https://user-images.githubusercontent.com/132835/39452713-357a1a40-4ca1-11e8-8d82-f771359e252f.png)

:/
